windows/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -216,6 +226,12 @@ void serverStartingMsg() {
 
 void checkServerThread()
 {
+	wchar_t msgString[120];

```

Comment:
This 120 character-limiting index seems unusual.  Although I think it's trivial for now but let's take note in case the notification will take longer than 120 characters.